blog/content/post/2022-04-12-talos2_2nd_cpu_and_testing.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Talos II - second CPU support and test automation
+abstract: 'Another post about our adventures with porting coreboot for Talos II.
+          This phase focused on enabling second CPU and its internal devices.
+          We also expanded our test suite.'
+cover: /covers/coreboot-logo.svg
+author: krystian.hebel
+layout: post
+published: true
+date: 2022-04-12
+archives: "2022"
+
+tags:
+  - coreboot
+  - firmware
+  - openpower
+categories:
+  - Firmware
+
+---
+
+Another post about our adventures with porting coreboot for Talos II. This phase
+focused on enabling second CPU and its internal devices. We also expanded our
+test suite.
+
+## Links to binaries and installation instructions
+
+TBD
```

Comment:
@krystian-hebel We already know the links. Maybe we can just link to the relevent section in the documentation?

Reply:
Added